### PR TITLE
update location of parenthesis for correct functionality

### PR DESCRIPTION
**Pull request by ros3bud**
@alexkwolfe or @cgrayson, LGTY?

### CODE REVIEW

File: spec/outbound-spec.coffee [hunk 1]
```diff
@@ -25,7 +25,7 @@ describe 'Outbound', ->
           mobile_opt_in: true
           credit_score: '720-900'
           company:
-            name: 'edu trainers    ' 
+            name:'edu trainers '
```
Comment (alexkwolfe) [inline spec/outbound-spec.coffee]: Is this relevant to the change?

**Comment by ros3bud** [comment]
Changing the amount of space there is not relevant/pertinent. What matters is that the trailing white is removed, regardless of how much or how little is there.
